recipes/ua-nodeset/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
```

Comment:
You're missing a test package.
It can be very trivial, e.g. testing an essential file.